### ISSUE TITLE
pipe data into mrpeek breaks mouse interactions

### ISSUE THREAD

**Issue by maxpietsch** [MRtrix3/mrpeek#22]
`mrcalc T1.mif 1200 -div - | bin/mrpeek - -axis 2 -scale 2`

click and drag:

![image](https://user-images.githubusercontent.com/10046944/85170674-08159400-b26e-11ea-938b-063aa8b87f4c.png)

the terminal remains unusable after ctrl+c.

**Comment by jdtournier** [comment]
This is going to be a hard one to fix... It'll require redirecting `stdin` after we've read the input filename back to the actual terminal, but before we set it to raw mode. I think I've seen it done before, I'll try to find it again.

The fact that ctrl-C has any effect whatsoever is because we've used the termios() call on a file handle that isn't actually the terminal... Since it's actually triggered the signal, it's completely expected that it would leave the terminal in an unusable state - see prior discussion we had about the need for signal handlers before we can safely handle `SIGINT` and `SIGSTOP`.
